CIScripts/Test/TestRunner.ps1

```diff
@@ -19,6 +19,10 @@
 function Run-TestScenarios {
     Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
 
+    if(-not $Sessions) {
+        return
```

Comment:
I think a warning should be printed here.

Reply:
I'm going to remove this early return (we abandoned the `map _ null = []` idea)